bin/setup

```diff
@@ -20,11 +19,11 @@ chdir APP_ROOT do
 
   # puts "\n== Copying sample files =="
   # unless File.exist?('config/database.yml')
-  #   cp 'config/database.yml.sample', 'config/database.yml'
+  #   FileUtils.cp 'config/database.yml.sample', 'config/database.yml'
   # end
 
   puts "\n== Preparing database =="
-  system! 'bin/rails db:setup'
+  system! 'bin/rails db:prepare'
```

Comment:
I wonder what was wrong with `setup` 🤔

Reply:
The new convention now seems to be using `db:prepare` which runs `db:setup` if the database doesn't exist, otherwise runs migrations: https://hint.io/blog/10-New-Things-in-Active-Record